fastlane/Fastfile

```diff
@@ -408,6 +408,19 @@ end
       dark_mode: options[:mode].to_s.downcase == "dark"
     )
   end
+
+  lane :create_promo_screenshots do |options|
+    unless Fastlane::Helper::GitHelper.has_git_lfs then
+      UI.user_error!("Git LFS not enabled – Unable to generate promo screenshots. Run `git lfs install && git lfs fetch && git lfs pull` to fix this.")
```

Comment:
I copied this [from the WooCommerce iOS repo](https://github.com/woocommerce/woocommerce-ios/blob/a9caacb9c738d363dee9c180733978d89d384055/fastlane/Fastfile#L506-L526).

@jkmassel , I'm not sure why this is required. But I can see `fastlane/appstroreres/assets` stores its masks with LFS. Good call, I'll do the same.